main.sh

```diff
@@ -46,25 +46,27 @@ else
   . "$dir/../@google-cloud/cloud-rad/generate-devsite.sh"
 fi
 
-# create docs.metadata, based on package.json and .repo-metadata.json.
-pip install -U pip
-python3 -m pip install --user gcp-docuploader
-python3 -m docuploader create-metadata \
-  --name=$name \
-  --version=$(cat package.json | json version) \
-  --language=$(cat .repo-metadata.json | json language) \
-  --distribution-name=$(cat .repo-metadata.json | json distribution_name) \
-  --product-page=$(cat .repo-metadata.json | json product_documentation) \
-  --github-repository=$(cat .repo-metadata.json | json repo) \
-  --issue-tracker=$(cat .repo-metadata.json | json issue_tracker)
-cp docs.metadata ./_devsite/docs.metadata
+if [[ -z "$NO_UPLOAD" ]]; then
```

Comment:
Could we flip this so we exit early if `NO_UPLOAD` is set? That way, we don't need to indent everything else.

https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow is relevant, but for Go.

Reply:
Done, thanks!